record-two/record.app.js

```diff
@@ -70,10 +70,12 @@
 
                     for (var i = 0; i < $rootScope.relatedReferences.length; i++) {
                         $rootScope.relatedReferences[i] = $rootScope.relatedReferences[i].contextualize.compactBrief;
-                        // We want to limit the number of values shown by default
-                        // Maybe have a chaise config option
+
+                        if ($rootScope.relatedReferences[i].display.defaultPageSize) {
+                            pageInfo.pageLimit = $rootScope.relatedReferences[i].display.defaultPageSize;
+                        }
```

Comment:
if the default is not specified, pageLimit should be set to 25, which is the recordset app default.